pyControl4/relay.py

```diff
@@ -0,0 +1,79 @@
+"""Controls Control4 Relay devices. These can include locks, and potentially other types of devices.
+"""
+
+
+class C4Relay:
+    def __init__(self, C4Director, item_id):
+        """Creates a Control4 Relay object.
+
+        Parameters:
+            `C4Director` - A `pyControl4.director.C4Director` object that
+            corresponds to the Control4 Director that the Relay is connected to.
+
+            `item_id` - The Control4 item ID of the Relay.
+        """
+        self.director = C4Director
+        self.item_id = item_id
+
+    async def getRelayState(self):
+        """Returns the current state of the relay."""
+        return await self.director.getItemVariableValue(self.item_id, "RelayState")
```

Comment:
@BrycesonKimura I know this is a old PR, but I'll be able to merge this after fixing this one issue.

Do you know what the potential return values of this function are? (e.g. what are the possible relay states?) We need to document this so we know how to develop relay integrations in the future.

Reply:
Not author but also have a C4 integrated door lock and have been following/lurking the project.

```python
relay = C4Relay(director, 56)
print(asyncio.run(relay.getRelayState()))
```

Gets me `0` when locked and `1` when unlocked.

Reply:
Sorry for the late response @lawtancool , I have become very busy with work and I don't have a Control4 System to test this on anymore, but mellis is correct. A relay just represents a binary state, on/off, open/close. The C4 locks are controlled by a relay, but a relay can be used for other devices, and it might not be limited to just locks.

Reply:
If you need any more help testing, or there is any more information I can provide, happy to help.

Reply:
Thanks @mellis and @BrycesonKimura for the help! I've added the return values in the code documentation.